steno/src/model/app_constants.h

```diff
@@ -0,0 +1,19 @@
+#ifndef KEYS_H
+#define KEYS_H
+
+#include <QDir>
+#include <QString>
+
+namespace Model
+{
+const QString CAMERA_CONFIGURATION = QDir::homePath();
+const QString MICROPHONE_CONFIGURATION =
+    QDir::homePath() + "/dev/workspace/rendezvous/steno/configs/odas/odas_16_mic.cfg";
+const QString ODAS_LIBRARY = QDir::homePath() + "/dev/lib/odas/bin/odaslive";
+const QString GOOGLE_SERVICE_ACCOUNT_FILE = QDir::homePath();
```

Comment:
we could remove the google service account variable since it will be in the transcription_api